### PR TITLE
Botan integration

### PR DESCRIPTION
**Pull request by ollmer**
Added module that allows quickly send bot events to Botan analytics service (http://botan.io). Module located in utils.

### CODE REVIEW

File: telegram/utils/botan.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                   'token={token}&uid={uid}&name={name}'
+
+    def __init__(self, token):
+        self.token = token
+
+    def track(self, message, event_name='event'):
+        try:
+            uid = message.chat_id
+        except AttributeError:
+            print('no chat_id in message')
```
Comment (jh0ker) [inline telegram/utils/botan.py]: Please use logging instead of print

Comment (ollmer) [inline telegram/utils/botan.py]: Done

File: tests/botan.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+class MessageMock(object):
+    chat_id = None
+
+    def __init__(self, chat_id):
+        self.chat_id = chat_id
+
+class BotanTest(BaseTest, unittest.TestCase):
+    """This object represents Tests for Botan analytics integration."""
+    token = os.environ.get('TOKEN')
```
Comment (jh0ker) [inline tests/botan.py]: This is the bot token from BotFather. Please use `BOTAN_TOKEN` to get the token for the botan.io API (we'll set it up)

Comment (ollmer) [inline tests/botan.py]: Done

**Comment by jh0ker** [comment]
Thanks a lot, Oleg :)

**Pull request merged**